backend/common.go

```diff
@@ -162,6 +162,9 @@ type PluginContext struct {
 	//
 	// Will only be set if request targeting a data source instance.
 	DataSourceInstanceSettings *DataSourceInstanceSettings
+
+	// Config is the configuration of the plugin instance.
```

Comment:
Nit. Global config something something? Plugin instance a bit unclear what it means/refer to